### PR TITLE
ops changes for post WCOSS changes

### PR DESCRIPTION
**Pull request by malloryprow**
<b>Note to developers: You must use this PR template!</b>

## Description of Changes

This brings in the changes from the WCOSS2 changes into a release branch for the emergency, accelerated change for EVS v1.0.13. See the changes in https://docs.google.com/document/d/1Kwd6WoDRYOE2n5ZbkoPkoFofRuAbWM572hdDdXli-qo/edit?tab=t.0.

## Developer Questions and Checklist
* Is this a high priority PR? If so, why and is there a date it needs to be merged by?
> We aren't running this release branch but it be good to have a release branch consistent with operations.
* Do you have any planned upcoming annual leave/PTO?
> Leave 11/27, 11/29; Federal Holiday 11/28 
* Are there any changes needed for when the jobs are supposed to run?
> No
  
- [ ] The code changes follow [NCO's EE2 Standards](https://www.nco.ncep.noaa.gov/idsb/implementation_standards/ImplementationStandards.v11.0.0.pdf).
- [ ] Developer's name is removed throughout the code and have used `${USER}` where necessary throughout the code.
- [ ] References the feature branch for `HOMEevs` are removed from the code.
- [ ] J-Job environment variables, COMIN and COMOUT directories, and output follow what has been [defined](https://docs.google.com/document/d/1JWg_4q80aYmmAoD21GFjp9R9y5-3w7WGM3-0HJk0Pjs/edit#heading=h.7ysbr191vzu4) for EVS.
- [ ] Jobs over 15 minutes in runtime have restart capability.
- [ ] If applicable, changes in the `dev/drivers/scripts` or `dev/modulefiles` have been made in the corresponding `ecf/scripts` and `ecf/defs/evs-nco.def`? 
- [ ] Jobs contain the approriate file checking and don't run METplus for any missing data.
- [ ] Code is using METplus wrappers structure and not calling MET executables directly.
- [ ] Log is free of any ERRORs or WARNINGs.

## Testing Instructions

NCO tested these changes. There will be another PR that brings the changes into develop that we can test. I can provide testing instructions if we wish to test this release branch.

### CODE REVIEW

File: ecf/scripts/plots/global_ens/jevs_global_ens_atmos_gefs_grid2obs_past90days_plots.ecf
Comment (AliciaBentley-NOAA) [inline ecf/scripts/plots/global_ens/jevs_global_ens_atmos_gefs_grid2obs_past90days_plots.ecf:7]: @malloryprow The corresponding dev driver for this .ecf script doesn't include the `exclhost` part. Do we know which version of this line is correct and make the dev driver and .ecf script match?

File: ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past31days_plots.ecf
Comment (AliciaBentley-NOAA) [inline ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past31days_plots.ecf:7]: @malloryprow This .ecf script line includes` shared`, but this is not included in the dev driver version. Do we know which version is correct? Can the dev drivers and .ecf versions match? Thanks!

File: ecf/scripts/plots/mesoscale/jevs_mesoscale_grid2obs_plots.ecf
Comment (PerryShafran-NOAA) [inline ecf/scripts/plots/mesoscale/jevs_mesoscale_grid2obs_plots.ecf:55]: @malloryprow I made this change in my current PR #603 .  I guess it doesn't matter if we make this change twice.

File: dev/drivers/scripts/plots/mesoscale/jevs_mesoscale_grid2obs_plots.sh
Comment (PerryShafran-NOAA) [inline dev/drivers/scripts/plots/mesoscale/jevs_mesoscale_grid2obs_plots.sh:29]: @malloryprow I made this change in my current PR #603 .

File: ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past90days_plots.ecf
Comment (AliciaBentley-NOAA) [inline ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past90days_plots.ecf:7]: @malloryprow This .ecf script line also includes `shared`, but this is not included in the dev driver version. Do we know which version is correct? Can the dev drivers and .ecf versions match? Thanks!

File: ecf/scripts/plots/global_ens/jevs_global_ens_atmos_gefs_grid2obs_past90days_plots.ecf
Comment (malloryprow) [reply ecf/scripts/plots/global_ens/jevs_global_ens_atmos_gefs_grid2obs_past90days_plots.ecf:7]: /lfs/h1/ops/prod/packages/evs.v1.0.13/ecf/scripts/plots/global_ens/jevs_global_ens_atmos_gefs_grid2obs_past90days_plots.ecf has `#PBS -l place=vscatter:exclhost,select=2:ncpus=88:mem=500GB` so we will got with that since that is what they decided to implement.

File: ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past31days_plots.ecf
Comment (malloryprow) [reply ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past31days_plots.ecf:7]: /lfs/h1/ops/prod/packages/evs.v1.0.13/ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past31days_plots.ecf has `#PBS -l place=vscatter:shared,select=2:ncpus=66:mem=500GB` so we will go with shared since that is what NCO implemented.

File: dev/drivers/scripts/plots/mesoscale/jevs_mesoscale_grid2obs_plots.sh
Comment (malloryprow) [reply dev/drivers/scripts/plots/mesoscale/jevs_mesoscale_grid2obs_plots.sh:29]: This PR isn't going into develop

File: ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past90days_plots.ecf
Comment (malloryprow) [reply ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past90days_plots.ecf:7]: /lfs/h1/ops/prod/packages/evs.v1.0.13/ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past90days_plots.ecf has `#PBS -l place=vscatter:shared,select=2:ncpus=66:mem=500GB`

Comment (malloryprow) [reply ecf/scripts/plots/cam/jevs_cam_href_grid2obs_ecnt_past90days_plots.ecf:7]: will update to that

**Comment by PerryShafran-NOAA** [comment]
@malloryprow Do you recommend that I test anything?

**Comment by AliciaBentley-NOAA** [comment]
@malloryprow I have reviewed this PR and think that it is likely good to merge after my handful of comments are addressed. One additional check that I'd like request before it is merged is that you do a `diff` of 1) this release (EVS v1.0.13) and 2) the ops version of EVS v1.0.13 on WCOSS2. The only differences should be the changes that we made to the dev driver scripts to match Wei's changes to the ops .ecf scripts. If there are more differences than that, we can talk about them. Thanks!

CC @AndrewBenjamin-NOAA

**Comment by malloryprow** [comment]
> @malloryprow I have reviewed this PR and think that it is likely good to merge after my handful of comments are addressed. One additional check that I'd like request before it is merged is that you do a `diff` of 1) this release (EVS v1.0.13) and 2) the ops version of EVS v1.0.13 on WCOSS2. The only differences should be the changes that we made to the dev driver scripts to match Wei's changes to the ops .ecf scripts. If there are more differences than that, we can talk about them. Thanks!
> 
> CC @AndrewBenjamin-NOAA

Yup, I did that before I submitted!

**Review by PerryShafran-NOAA — APPROVED** [review]
All seems to be in order.  I approve for merge.

**Review by PerryShafran-NOAA — APPROVED** [review]
Approved for merge.

**Comment by malloryprow** [comment]
Do we want to merge this?

**Comment by AliciaBentley-NOAA** [comment]
Yes! If the only diffs from ops EVS v1.0.13 were the dev driver changes, I think we're good to go here!

**Comment by PerryShafran-NOAA** [comment]
@malloryprow I think we can.  

Do you want to change the 2nd reviewer from Andrew to Alicia?

**Comment by malloryprow** [comment]
I added both since it touches both regional and global, but Andrew is out some of today and Alicia approved!

**Comment by AliciaBentley-NOAA** [comment]
@malloryprow Do you want to switch Andrew to me as the second reviewer so this can be approved and merged this morning? CC @PerryShafran-NOAA

**Comment by malloryprow** [comment]
Hold PR as NCO is finding more problematic jobs on Cactus. We may need more changes.

**Comment by malloryprow** [comment]
The changes we were holding off on NCO put them in a different version. This PR is specifically for the ops EVS version v1.0.13. We can merge this.

**Comment by PerryShafran-NOAA** [comment]
@malloryprow @AliciaBentley-NOAA I had already approved this long ago.  Needs the 2nd approval.

**Comment by AliciaBentley-NOAA** [comment]
> @malloryprow I have reviewed this PR and think that it is likely good to merge after my handful of comments are addressed. One additional check that I'd like request before it is merged is that you do a `diff` of 1) this release (EVS v1.0.13) and 2) the ops version of EVS v1.0.13 on WCOSS2. The only differences should be the changes that we made to the dev driver scripts to match Wei's changes to the ops .ecf scripts. If there are more differences than that, we can talk about them. Thanks!
> 
> CC @AndrewBenjamin-NOAA

Looks like my review from 3 weeks ago didn't get marked as official! I have gone back and made sure that the handful of comments I pointed out were fixed, and they were. I approve this PR to be merged.

**Review by AliciaBentley-NOAA — APPROVED** [review]
I have gone back and made sure that the handful of comments I pointed out three weeks ago were fixed, and they were! I approve this PR to be merged.

**Pull request merged**
